packages/cli/src/commands/deploy/args.ts

```diff
@@ -89,14 +36,15 @@ export const help = () => `
 
   ${chalk.gray('–')} Deploy with Environment Variables
 
-    ${chalk.cyan(
-      `$ ${getPkgName()} -e NODE_ENV=production -e SECRET=@mysql-secret`
-    )}
+    ${chalk.cyan(`$ ${getPkgName()} -e NODE_ENV=production`)}
+
+  ${chalk.gray('–')} Deploy with prebuilt outputs
 
-  ${chalk.gray('–')} Show the usage information for the sub command ${chalk.dim(
-  '`list`'
-)}
+    ${chalk.cyan(`$ ${getPkgName()} build`)}
+    ${chalk.cyan(`$ ${getPkgName()} deploy --prebuilt`)}
 
-    ${chalk.cyan(`$ ${getPkgName()} help list`)}
+  ${chalk.gray('–')} Save reference to Deployment URL
```

Comment:
"Reference" is kinda vague. How about:

```suggestion
  ${chalk.gray('–')} Write Deployment URL to a file
```